src/main/java/com/example/accommodationbookingapp/dto/booking/BookingResponseDto.java

```diff
@@ -0,0 +1,17 @@
+package com.example.accommodationbookingapp.dto.booking;
+
+import com.example.accommodationbookingapp.model.Accommodation;
+import com.example.accommodationbookingapp.model.Booking;
+import com.example.accommodationbookingapp.model.User;
+import java.time.LocalDate;
+import lombok.Data;
+
+@Data
+public class BookingResponseDto {
+    private Long id;
+    private LocalDate checkInDate;
+    private LocalDate checkOutDate;
+    private Accommodation accommodationId;
+    private User userId;
```

Comment:
Should be Long if you want to return ID, or accomodationresponsedto if you want to have dto inside dto. but i guess id would be simpler